src/components/Animation.tsx

```diff
@@ -0,0 +1,196 @@
+import React from 'react';
+import Gallery from './Gallery';
+
+interface State {
+  animationName: string,
+  vector: number,
+  from: number,
+  to: number,
+}
+
+interface Props {
+  infiniteAuto: boolean,
+  infinite: boolean,
+  itemWidth: number,
+  frameSize: number,
+  duration: number,
+  images: string[];
+  step: number;
+}
+
+class Battons extends React.Component<Props, State> {
```

Comment:
```suggestion
class Buttons extends React.Component<Props, State> {
```
Typo in 'buttons'. Also, file should be named just like component